sway/commands/shadow_color.c

```diff
@@ -17,6 +17,9 @@ struct cmd_results *cmd_shadow_color(int argc, char **argv) {
 		return cmd_results_new(CMD_INVALID, "Invalid %s color %s",
 				"shadow_color", argv[0]);
 	}
+	if (memcmp(config->shadow_color, config->shadow_inactive_color, sizeof(config->shadow_color)) == 0) {
+		color_to_rgba(config->shadow_inactive_color, color);
+	}
```

Comment:
I think this would result in shadow_inactive_color being set twice (once here and then again in the shadow_inactive_color command file), so this can be omitted

Reply:
That is there so if you only set the shadow_color (and not the shadow_inactive_color) both of them are set to your custom one like right now, so that the changes are backward compatible. It does mean that it is set twice but the alternative is to make everyone change the configs to also add shadow_inactive_color even if they want shadows to be uniform. I'm ok changing it either way though, so please tell me!

Reply:
Also if shadow_inactive_color is already set that is ignored, that's what the memcmp is for, so you can define them in any order.

Reply:
Ah got it, misinterpreted the comparison